v2/internal/x509util/util.go

```diff
@@ -35,3 +35,19 @@ func CertsEqual(a, b []*x509.Certificate) bool {
 
 	return true
 }
+
+func RawCertsFromCerts(certs []*x509.Certificate) [][]byte {
```

Comment:
nit: it sound a little redundant, maybe we can rename it to RawCerts()/ConcatRawCerts?

Reply:
Hmm. I'd be open to a change. Since this is an internal function we have some latitude about changing it later. Are you ok if we merge this and think on it?

Reply:
sure